cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1673,6 +1673,26 @@ private static void validateOauth(KafkaListenerAuthenticationOAuth oAuth, String
             log.error("{}: The refresh interval has to be at least 60 seconds shorter then the expiry interval specified in `jwksExpirySeconds`", listener);
             throw new InvalidResourceException(listener + ": The refresh interval has to be at least 60 seconds shorter then the expiry interval specified in `jwksExpirySeconds`");
         }
+
+        if (oAuth.isNotJwt()) {
+            if (oAuth.getJwksEndpointUri() != null) {
```

Comment:
Should we also check these properties for emptiness, just to be safe?

Reply:
Not following here. The `accessTokenIsJwt` is `boolean` - can't be empty. `jwksEndpointUri` is null unless set in which case it won't be null.